src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/satsendring/BarnetilsynSatsendringService.kt

```diff
@@ -93,7 +95,9 @@ class BarnetilsynSatsendringService(
                 barn = it.andel.barn,
                 utgifter = it.andel.utgifter.toInt(),
                 erMidlertidigOpphør = false,
-                sanksjonsårsak = null
+                sanksjonsårsak = null,
+                periodetype = PeriodetypeBarnetilsyn.ORDINÆR,
+                aktivitetstype = AktivitetstypeBarnetilsyn.I_ARBEID
```

Comment:
Hvorfor har disse defaultverdier? Burde disse bruke verdier fra selve `it` her? Fra `List<AndelHistorikkDto>`

Reply:
Fixd [7ed490d](https://github.com/navikt/familie-ef-sak/pull/2021/commits/7ed490d4044fe39cd0797c866a10758dc9a10344)